package.json

```diff
@@ -23,8 +23,12 @@
     "typescript": "3.2.4"
   },
   "devDependencies": {
+    "@babel/core": "7.2.2",
```

Comment:
When I remove this, storybook loads fine. I think it only needs `babel-loader`.

Reply:
Correct, we don't need this. Thanks!